### PR TITLE
Removed unused label styles

### PR DESCRIPTION
**Pull request by mperrotti**
Are these lines safe to delete? They're also in `meetup-web-compoments` in `assets/scss/components/_forms.scss`

### CODE REVIEW

File: scss/reset/_forms.scss [hunk 1]
```diff
@@ -81,15 +81,6 @@ label {
 	}
 }
 
-.label--field {
-	padding-bottom: $space-half;
-	display: inline-block;
-}
-
-.label--minor {
```
Comment (akdetrick) [inline scss/reset/_forms.scss]: `label--minor` is being used by `Checkbox` in meetup-web-components

File: scss/reset/_forms.scss [hunk 2]
```diff
@@ -81,15 +81,6 @@ label {
 	}
 }
 
-.label--field {
```
Comment (akdetrick) [inline scss/reset/_forms.scss]: `label--field` is the class applied to `label` elements in `meetup-web-components`

Comment (mperrotti) [inline scss/reset/_forms.scss]: Yes, but these same styles are in `assets/scss/components/_forms.scss` in `meetup-web-components`

File: scss/reset/_forms.scss (hunk 1 above)
Comment (mperrotti) [inline scss/reset/_forms.scss]: Yes, but these same styles are in `assets/scss/components/_forms.scss` in `meetup-web-components`

**Comment by akdetrick** [comment]
Yep, these are totally duplicated in web components; LGTM

**Pull request merged**